packages/harmony/src/assets/styles/shadows.css

```diff
@@ -6,6 +6,7 @@ root {
   --harmony-shadow-near: 0px 2px 4px 0px rgba(0, 0, 0, 0.08), 0px 0px 6px 0px rgba(0, 0, 0, 0.02);
   --harmony-shadow-mid: 0px 4px 8px 0px rgba(0, 0, 0, 0.06), 0px 0px 4px 0px rgba(0, 0, 0, 0.04);
   --harmony-shadow-far: 0px 8px 16px 0px rgba(0, 0, 0, 0.08), 0px 0px 4px 0px rgba(0, 0, 0, 0.04);
+  --harmony-shadow-emphasis: 0px 1.34018px 8px 0px rgba(0, 0, 0, 0.20), 0px 6px 15px 0px rgba(0, 0, 0, 0.10);
 
-  --harmony-drop-shadow: drop-shadow(0px 1.3401784896850586px 8px rgba(0, 0, 0, 0.20)) drop-shadow(0px 6px 15px rgba(0, 0, 0, 0.10));
+  --harmony-drop-shadow: drop-shadow(0px 1.34018px 8px rgba(0, 0, 0, 0.20)) drop-shadow(0px 6px 15px rgba(0, 0, 0, 0.10));
```

Comment:
missed opportunity to make it digits of 🥧

also is there a reason not to use `1.3px`?

Reply:
just copied the value over from figma. It seemed really specific so I didn't want to touch it